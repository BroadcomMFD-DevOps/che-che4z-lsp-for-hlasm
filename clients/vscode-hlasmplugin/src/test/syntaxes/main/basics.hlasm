LABEL INSTR OP1
LABEL INSTR OP1,OP2                                                             
LABEL INSTR OP1,OP2

LABEL INSTR OP1,OP2                                                    -12345678
               REMARKS                                                  12345678
LABEL INSTR OP1,OP2,                                                   -
               OP3,                                                    -
               OP4                                                      12345678
LABEL INSTR OP1,OP2,                                                   -12345678
               OP3,                                                    -12345678
               OP4                                                      12345678

LABEL INSTR OP1,OP2,                                                   -12346578
               OP3

LABEL INSTR OP1,OP2                                                    -
               REMARKS,                                                -
               REMARKS                                                  12345678

LABEL INSTR OP1,OP2    REMARKS                                         -12345678
               REMARKS,                                                -12346578
               REMARKS                                                  12345678

LABEL INSTR OP1,OP2,      REMARKS  REMARKS                             -
               OP3, REMARKS                                            -12345678
               OP4                                                     -12345678
               REMARKS

LABEL INSTR OP1,OP2,   REMARKS,                                        -12345678
               OP3,OP4,       RERMARKS                                 -12346578
               OP5,OP6        RERMARKS                                 -12346578
               REMARKS,                                                -12346578
               REMARKS                                                 -12346578
               REMARKS                                          REMARKS-12346578
               REMARKS                                                  12345678

LABEL INSTR OP1,OP2,   REMARKS                                         -12345678
               OP3,OP4,       REMARKS                                  -12346578
               OP5,OP6                                                 -12346578
               REMARKS                                                 -12346578
               REMARKS                                                  12345678
               


LABEL INSTR OP1,OP2    REMARKS,                                        -12345678
               REMARKS,REMARKS,       REMARKS                          -12346578
               REMARKS,REMARKS                                         -12346578
               REMARKS            REMARKS                              -12346578
               REMARKS                                                  12345678

LABEL INSTR      OP1                                                   -
                                                                       -
               REMARKS,                                                -
                                                                       -
               REMARKS REMARKS                                         -
               REMARKS REMARKS                                         -
               REMARKS REMARKS                                          12345678

LABEL INSTR OP1,OP2    REMARKS                                         -        
INVALID,                                                               -12346578 
               REMARKS                                                  12345678


LABEL INSTR                                                            -12345678
               OP1,OP2,   REMARKS                                      -12345678
               OP3,    REMARKS                                         -12346578
               OP4                                                      12345678

LABEL INSTR                                                            -12345678
               OP1 REMARKS,   REMARKS                                  -12345678
               REMARKS,    REMARKS                                     -12346578
               REMARKS                                                  12345678
