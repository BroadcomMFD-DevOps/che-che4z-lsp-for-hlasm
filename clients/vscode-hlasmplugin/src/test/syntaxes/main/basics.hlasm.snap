>LABEL INSTR OP1
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
>LABEL INSTR OP1,OP2                                                             
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>LABEL INSTR OP1,OP2
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
>
>LABEL INSTR OP1,OP2                                                    -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                                  12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>LABEL INSTR OP1,OP2,                                                   -
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^ source.hlasm
#                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               OP3,                                                    -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^ source.hlasm
#                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               OP4                                                      12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>LABEL INSTR OP1,OP2,                                                   -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^ source.hlasm
#                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP3,                                                    -12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^ source.hlasm
#                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP4                                                      12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>
>LABEL INSTR OP1,OP2,                                                   -12346578
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^ source.hlasm
#                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP3
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
>
>LABEL INSTR OP1,OP2                                                    -
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               REMARKS,                                                -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               REMARKS                                                  12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>
>LABEL INSTR OP1,OP2    REMARKS                                         -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^^^^ source.hlasm
#                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS,                                                -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                                  12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>
>LABEL INSTR OP1,OP2,      REMARKS  REMARKS                             -
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^ source.hlasm
#                     ^^^^^ source.hlasm
#                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               OP3, REMARKS                                            -12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^ source.hlasm
#                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP4                                                     -12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^ source.hlasm comment.hlasm
>
>LABEL INSTR OP1,OP2,   REMARKS,                                        -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^ source.hlasm
#                     ^^ source.hlasm
#                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP3,OP4,       RERMARKS                                 -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm punctuation.definition.tag.hlasm
#                      ^ source.hlasm
#                       ^ source.hlasm
#                        ^^^^^^ source.hlasm
#                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP5,OP6        RERMARKS                                 -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm punctuation.definition.tag.hlasm
#                      ^^^^^^^^ source.hlasm
#                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS,                                                -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                                 -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                          REMARKS-12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                                  12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>
>LABEL INSTR OP1,OP2,   REMARKS                                         -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^ source.hlasm
#                     ^^ source.hlasm
#                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP3,OP4,       REMARKS                                  -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm punctuation.definition.tag.hlasm
#                      ^ source.hlasm
#                       ^ source.hlasm
#                        ^^^^^^ source.hlasm
#                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP5,OP6                                                 -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm punctuation.definition.tag.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                                 -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                                  12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               
>
>
>LABEL INSTR OP1,OP2    REMARKS,                                        -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^^^^ source.hlasm
#                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS,REMARKS,       REMARKS                          -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS,REMARKS                                         -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS            REMARKS                              -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                                  12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>
>LABEL INSTR      OP1                                                   -
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^^^^^^ source.hlasm
#                 ^^^ source.hlasm punctuation.definition.tag.hlasm
#                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>                                                                       -
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               REMARKS,                                                -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>                                                                       -
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               REMARKS REMARKS                                         -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               REMARKS REMARKS                                         -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               REMARKS REMARKS                                          12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>
>LABEL INSTR OP1,OP2    REMARKS                                         -        
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^ source.hlasm
#            ^^^ source.hlasm punctuation.definition.tag.hlasm
#               ^ source.hlasm
#                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^^^^ source.hlasm
#                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>INVALID,                                                               -12346578 
#^^^^^^^^ source.hlasm invalid.illegal
#        ^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                                  12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>
>
>LABEL INSTR                                                            -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP1,OP2,   REMARKS                                      -12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^ source.hlasm punctuation.definition.tag.hlasm
#                      ^ source.hlasm
#                       ^ source.hlasm
#                        ^^ source.hlasm
#                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP3,    REMARKS                                         -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^ source.hlasm
#                    ^^^ source.hlasm
#                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP4                                                      12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>
>LABEL INSTR                                                            -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^ source.hlasm
#      ^^^^^ source.hlasm entity.name.function.hlasm
#           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP1 REMARKS,   REMARKS                                  -12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS,    REMARKS                                     -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARKS                                                  12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>